.github/workflows/format-code.yaml

```diff
@@ -0,0 +1,25 @@
+name: Check code
+on:
+  pull_request:
+    branches: [main]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  main:
+    runs-on: ubuntu-22.04
```

Comment:
Can we run this on 24.04?

Reply:
yeah, think this should be possible. Will add that once working.

Reply:
now added latest tag